packages/ui/src/2_molecules/TableBase/components/TableRow/TableRow.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useCallback, useState } from 'react';
+
+import classNames from 'classnames';
+
+import { noop } from '../../../../utils';
+import { RowObject, ColumnOptions } from '../../TableBase.types';
+import styles from './TableRow.module.css';
+
+type TableRowProps<RowType extends RowObject> = {
+  columns: ColumnOptions<RowType>[];
+  row: RowType;
+  index: number;
+  onRowClick?: (row: RowType) => void;
+  dataAttribute?: string;
+  isClickable?: boolean;
+};
+
+export const TableRow = <RowType extends RowObject>({
+  columns,
+  row,
+  index,
+  onRowClick = noop,
+  dataAttribute,
+  isClickable,
+}: TableRowProps<RowType>) => {
+  const [isSelected, setIsSelected] = useState(false);
+  const onClick = useCallback(() => {
+    setIsSelected(prevValue => !prevValue);
+    onRowClick?.(row);
+  }, [onRowClick, row]);
+
+  return (
+    <>
+      <tr
+        key={index}
+        className={classNames(styles.row, {
+          [styles.clickable]: isClickable,
+          [styles.active]: isClickable && isSelected,
+        })}
+        onClick={onClick}
+        data-layout-id={`${dataAttribute}-row-${index}`}
+      >
+        {columns.map(column => (
+          <td
+            key={column.id.toString()}
+            className={classNames(
+              column.align && [styles[`text${column.align}`]],
+            )}
+          >
+            {column.cellRenderer
+              ? column.cellRenderer(row, column.id)
+              : row[column.id]}
+          </td>
+        ))}
+      </tr>
+      <tr className={styles.spacer}></tr>
```

Comment:
wouldn't it be simpler to give bottom padding to the ``table tr -> th`` ?

Reply:
Yeah, I wish it was that simple :D But this is a table - `padding` is applied only in the cell itself and `margin` is ignored. I don't like this solution either but I could choose between more hacky and less hacky options.